ccd-definition/CaseEventToFields/createOrder.json

```diff
@@ -54,6 +54,48 @@
     "FieldShowCondition": "orderTypeAndDocument=\"DO_NOT_SHOW\"",
     "ShowSummaryChangeOption": "N"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "singleCareOrder_label",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "SingleCareOrder",
+    "PageLabel": " ",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" AND singleCareOrder_label!=\"\"",
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "multipleCareOrder_label",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "MultipleCareOrders",
+    "PageLabel": " ",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" AND multipleCareOrder_label!=\"\"",
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/validate-order/care-orders-selection/mid-event",
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "careOrderSelector",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "COMPLEX",
```

Comment:
That's what we discussed - not sure if it needs to be complex, if yes why? 🤔

Reply:
No it is not needed to be complex. Done